src/networks/neutron/mainnet/tokens.ts

```diff
@@ -789,8 +789,8 @@ export const tokens: (Omit<Token, "sources"> & {
     decimals: 6,
   },
   {
-    name: "nobleUSDC/NTRN 0d Vault Token",
```

Comment:
Didn't we say to remove the Unlocked part from the vt name?

Reply:
you're right.  in the new pr, i've removed 0d and "unlocked" from the existing ones as well.